src/status_im/contexts/wallet/add_account/create_account/utils.cljs

```diff
@@ -2,28 +2,40 @@
   (:require [status-im.constants :as constants]))
 
 (defn first-derived-account
-  [account-data]
-  (-> account-data :derived first val))
+  [account-data keypair-type]
+  (if (= keypair-type :seed)
```

Comment:
this is definitely right for all other types? 🤔 
Not saying it's not, just double checking :)

Reply:
I think I noticed when the `keypair-type` is `:key` (for private key) there wasn't any derived accounts in the account data. I'm not sure about the technical reasons, but from what I understood, private keys will not be able to have derived accounts.